src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -347,13 +371,46 @@ public function render()
                     }
                     if (isset($row[$i])) {
                         $rows[$i][] = $row[$i];
-                    } elseif ($rows[$i][0] instanceof TableCell && $rows[$i][0]->getColspan() >= 2) {
+                    } elseif ($isCellWithColspan($rows[$i][0])) {
                         // Noop, there is a "title"
                     } else {
                         $rows[$i][] = null;
                     }
                 }
             }
+        } elseif ($vertical) {
+            $maxHeaderLength = max(array_map(static function (string $header) {
+                return mb_strlen($header);
+            }, $this->headers[0] ?? ['']));
```

Comment:
Something like this?
```suggestion
            $maxHeaderLength = array_reduce($this->headers[0] ?? [], static fn (int $max, string $header) => max($max, mb_strlen($header)), 0);
```

Reply:
done